ui/challenge/src/view.ts

```diff
@@ -136,7 +136,8 @@ function renderUser(u?: ChallengeUser): VNode {
     class: { online: !!u.online }
   }, [
     h('i.line' + (u.patron ? '.patron' : '')),
-    h('name', (u.title ? u.title + ' ' : '') + u.name + ' (' + rating + ')')
+    h('name', (u.title ? u.title + ' ' : '') + u.name + ' (' + rating + ') '),
+    h('signal', Array.apply(null, {length: u.lag || 0}).map(() => h('i')))
```

Comment:
oh, no. The client can render it. But that's not the correct way. Checkout the HTML of a signal with 3/4 bars.